src/did/sidetree/client.rs

```diff
@@ -0,0 +1,50 @@
+use http::StatusCode;
+
+// This type isn't implement std::error::Error because of conflicting
+// implementations
+#[derive(Debug)]
+pub enum HttpError {
+    Inner(anyhow::Error),
+}
+
+impl<E> From<E> for HttpError
+where
+    E: std::error::Error + Send + Sync + 'static,
+{
+    fn from(e: E) -> Self {
+        Self::Inner(anyhow::Error::new(e))
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct SidetreeHttpClientResponse {
+    pub(crate) status_code: StatusCode,
+    pub(crate) body: String,
+}
+
+#[derive(Debug, thiserror::Error)]
+pub enum SidetreeResponseInitializationError {
```

Comment:
Can you tell me what `Initialization` means?

Reply:
@yokishava 
It's mean that `SidetreeResponse::new` failed.
If you have any other good name suggestions, please let me know.

Reply:
@yokishava 
If you have a good name, please comment here.
I'll fix it in the next PR.